app/api/users/specs/users.spec.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable max-lines */
 /** @format */
```

Comment:
Please remove the /** @format */ comment, is deprecated and we remove it from every file we modify.

Reply:
Done